config/environments/development.rb

```diff
@@ -36,6 +36,8 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+
```

Comment:
This doesn't look right, and I think could be removed if this branch is updated with the latest master branch.  Jamie has added Mailer configuration options.

Reply:
This actually sets that hostname to use in the link that gets emailed to the user, e.g. `http://localhost:3000/signin?token=xyz`. I agree it felt strange to put there, but it's in the passwordless docs and fails if it's not set. Perhaps @jamiebuckley 's updates supercede this.

Reply:
Captured in trello task